.github/workflows/demo-more-modules.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build more-modules app
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# Minimum permissions required by skip-duplicate-actions
```

Comment:
I don't understand what this does. What is "skip-duplicate-actions"?

Reply:
Will remove. Remnant of something I used to use.